### ISSUE TITLE
Language identifiers for langString are lowercase when imported from eCl@ss, uppercase in dropdowns.

### ISSUE THREAD

**Issue by JensPfeifle** [admin-shell/aasx-package-explorer#23]
**Describe the bug**
The langString elements of ConceptDescriptions imported from eCl@ss XML files are lowercase ('en'). The selection dropdowns of the editor suggest uppercase values ('EN'). This can cause a mismatch in the XML, which is case sensitive.

**Screenshots**
![image](https://user-images.githubusercontent.com/23027708/67617054-91f48680-f7df-11e9-96a4-194f9caa4b14.png)

**Version:**
Build 20190930, Version 0.9.6

**Comment by BirgitBoss** [comment]
In this case it is not the eCl@ss format who is the master but the metamodel of the AAS. The metamodel of AAS follows the l**angString specification of RDF**. (Although eCl@ss seems also to follow the same rules).
See:
https://www.w3.org/TR/rdf-schema/#ch_langstring
language tag: https://tools.ietf.org/html/bcp47#section-2.2.9
Examples:
https://tools.ietf.org/html/bcp47#appendix-A

   Simple language subtag:

      de (German)

      fr (French)

      ja (Japanese)

      i-enochian (example of a grandfathered tag)

   Language subtag plus Script subtag:

      zh-Hant (Chinese written using the Traditional Chinese script)

      zh-Hans (Chinese written using the Simplified Chinese script)

      sr-Cyrl (Serbian written using the Cyrillic script)

      sr-Latn (Serbian written using the Latin script)


So starting with small letter is correct. Of course it does not really matter was is shown in the editor as long as the export is correct. But you are rigth: it may confuse.

**Comment by JensPfeifle** [comment]
Good to see you here, Dr. Boss! Thanks for the clarification and examples.

The AASX PackageExplorer exports whatever was specified in the text box, so if uppercase was entered there, that would end up in the XML. In my opinion that is not a problem, but that the Explorer suggests uppercase for the langString is confusing. To get the correct string (as you described above) the user needs to ignore the uppercase suggestions in the dropdown-menu and manually enter the lowercase langString. This is what I did in the screenshot in the original issue description. 

This issue is based on build 20190930, if this has been corrected in the current builds, it can be closed as resolved.
